cli/tests/integration/shared_recovery/mod.rs

```diff
@@ -0,0 +1,56 @@
+use crate::utils::*;
+use libparsec::LocalDevice;
+use std::sync::Arc;
+
+use crate::testenv_utils::DEFAULT_DEVICE_PASSWORD;
+
+mod create;
+mod delete;
+mod info;
+mod list;
+
+fn setup_shared_recovery(
+    alice: &Arc<LocalDevice>,
+    bob: &Arc<LocalDevice>,
+    toto: &Arc<LocalDevice>,
+) {
+    crate::assert_cmd_success!(
+        with_password = DEFAULT_DEVICE_PASSWORD,
+        "shared-recovery",
+        "info",
+        "--device",
+        &alice.device_id.hex()
+    )
+    .stdout(predicates::str::contains(format!(
+        "Shared recovery {RED}never setup{RESET}"
+    )));
+    crate::assert_cmd_success!(
+        with_password = DEFAULT_DEVICE_PASSWORD,
+        "shared-recovery",
+        "create",
+        "--device",
+        &alice.device_id.hex(),
+        "--recipients",
+        &bob.human_handle.email(),
+        &toto.human_handle.email(),
+        "--weights",
+        "1",
+        "1",
+        "--threshold",
+        "1"
+    )
+    .stdout(predicates::str::contains(
+        "Shared recovery setup has been created",
+    ));
+
+    crate::assert_cmd_success!(
+        with_password = DEFAULT_DEVICE_PASSWORD,
+        "shared-recovery",
+        "info",
+        "--device",
+        &alice.device_id.hex()
+    )
+    .stdout(predicates::str::contains(format!(
+        "Shared recovery {GREEN}set up{RESET}"
+    )));
```

Comment:
Could this be done without using the CLI ?

Reply:
Personally I'd rather have more CLI testing than less, even though this test is meant to test another command than `shared-recovery info`.

I understand this is a trade-off between unit tests being more "unit" and more test coverage, and I tend to choose the latter.